CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,149 @@
+<div id="top"></div>
+
+<br />
+
+<div align="center">
+    <a href="https://github.com/nim-works">
+        <img src="https://raw.githubusercontent.com/nim-lang/assets/master/Art/logo-crown.png" height="80px"/>
+    </a>
+    <h1 align="center">Code of Conduct</h3>
+    <p align="center">
+    Last Revision - 22 Dec 2021
+    </p>
+</div>
+
+<br />
+
+### Purpose
+
+
+We want to make sustainable technology, this requires a community that is
+architected along the same principle. The Code of Conduct (CoC) is a key tool
+in ensuring a community that is diverse in experience, cohesive in practice,
+and growing in membership [^1] [^2]. We welcome feedback.
+
+<details><summary><b>Show more</b></summary><br />
+
+We empathize with serious concerns that CoC may represent a loss of freedom of
+speech, inhibit personal expression, or bring unnecessary discussion to what
+should be the development of technology [^3].
+
+To these concerns, we say that evidence-based research has shown that Codes
+of Conduct are instrumental to achieving and sustaining inclusive communities
+which promote diverse sets of skills and perspectives [^1] [^2] [^3] [^4]. Meritocracy is not
+sustainable outside of interactions between absolute equals, including cultural
+norms and understandings [^2].
+
+</details>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Scope
+
+This Code of Conduct applies to spaces both online and offline.
+
+<details>
+
+Online spaces managed by the Nim-Works organization will be held to the
+following standards. Spaces that represent our users, or where users are
+otherwise representing Nim-Works, may still be held accountable to these
+standards depending upon context and official status of the representative.
+
+Offline spaces where members interact, including but not limited to
+conferences, will be similarly held to the standards below.
+
+</details>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Code of Conduct
+
+Outlined below are the main values of the CoC, each of which can be expanded to
+show contributing rules and themes.
+
+1. <details><summary>We are an open and inclusive community, so I should behave in a manner which is respectful of others without discrimination.</summary><br />
+
+   1. Discrimination against an individual(s) or group(s) damages the community and attacks the goals of our contributions; we specifically will not discriminate based on:
+      1. Virtue of race or ethnicity; country of origin, or country of residence; gender(s), biological sex or sexual orientation; age or experience; economic standing or education; disability - physical or otherwise. Not listed are not excluded.
```

Comment:
```suggestion
   1. Discrimination against an individual(s) or group(s) damages the community and attacks the goals of our contributions; we specifically will not discriminate based on, but not limited to:
      1. Virtue of race or ethnicity; country of origin, or country of residence; gender(s), biological sex or sexual orientation; age or experience; economic standing or education; disability - physical or otherwise.
```

Flow better this way IMO.